Andlet/Andlet/Views/Helpers/Heading.swift

```diff
@@ -41,28 +41,34 @@ struct Heading: View {
                 }
             } else {
                 // Placeholder si no hay imagen disponible
-                Circle()
-                    .fill(Color.gray.opacity(0.3))
-                    .frame(width: 67, height: 67)
+                Image("Icon")
+                                    .resizable()
+                                    .frame(width: 67, height: 67)
+                                    .clipShape(Circle())
+                                    .shadow(radius: 5)
             }
         }
         .padding(.horizontal)
         .padding(.bottom, 3)
         .padding(.top, 5)
         .onAppear {
-                    // Si quieres ver el photoURL, lo puedes imprimir aquí
-                    if let photoURL = currentUser?.photoURL {
-                        print("AQUI ESTA LA FOTO")
-                        print(photoURL)
+            printUserDetails()
                     }
-                    if let user = currentUser {
-                            print("Información completa del usuario:")
-                            print(user)
-                        }
                 }
-    }
-}
-
+                
+                // Función para imprimir detalles del usuario
+                func printUserDetails() {
+                    if let currentUser = currentUser {
+                        print("Nombre de usuario: \(currentUser.displayName ?? "Desconocido")")
+                        print("URL de la foto: \(currentUser.photoURL?.absoluteString ?? "No hay foto disponible")")
+                        
+                        // Verifica si puedes acceder a más detalles
+                        print("Información completa del usuario: \(currentUser)")
+                    } else {
+                        print("No hay un usuario autenticado.")
+                    }
+                }
+            }
```

Comment:
Should we keep this function ?

Reply:
We could keep it just in case we need to confirm that information, now that we are sure it works, we don't need it